Fixes #2783

Ayutac opened the pull request:
Needs to be tested still. Goal: changed blasting to use half as much as time as regular furnace for same item.

EDIT: Tested by ErrorSys and debugged what they found. Can be merged now.

Commit:
Fixes #2783 hopefully

Needs to be tested still. Goal: changed blasting to use half as much as time as regular furnace for same item.

ErrorSys commented:
Compiles and works as intended, although I did have to specify experience on my end before IDEA would compile. Probably just on my end though as it should be an implied integer anyways.

Commit:
Fixed bug in commit thanks to ErrorSys

forgot that order of optional arguments is important and that int can be cast to float automatically
